packages/material-ui/src/Popper/Popper.test.js

```diff
@@ -123,6 +123,22 @@ describe('<Popper />', () => {
     });
   });
 
+  describe('prop: popperOptions', () => {
+    const popperProps = {
+      onCreate: spy(),
+      onUpdate: spy(),
+    };
+
+    it('should correctly pass down lifecycle callbacks to popperjs', () => {
+      const wrapper = mount(<Popper {...defaultProps} popperOptions={popperProps} />);
+      const instance = wrapper.find('Popper').instance();
```

Comment:
We will soon migrate the Popper component to the hook API. We will no longer be able to dive into the private properties of the component. The simplest option is probably not to test it.

Reply:
I also prefer tests to not call functions directly but through their public APIs i.e. In normal use

Reply:
@alltom Do you have a better idea than removing the test?

Reply:
I'm happy to remove it - just don't want to impact test coverage 😯! I'll have a look in the morning and see if I can test it another way, or remove the test 👍